packages/react-components/theme-designer/src/components/Sidebar/EditTab.tsx

```diff
@@ -41,32 +41,25 @@ const useStyles = makeStyles({
 export interface EditTabProps {
   sidebarId: string;
   dispatchState: React.Dispatch<DispatchTheme>;
+  formState: CustomAttributes;
+  setFormState: React.Dispatch<CustomAttributes>;
 }
 
 export const EditTab: React.FC<EditTabProps> = props => {
   const styles = useStyles();
 
-  const { sidebarId, dispatchState } = props;
+  const { sidebarId, dispatchState, formState, setFormState } = props;
```

Comment:
I don't think that the form state should move out of this tab. this was causing unnecessary re-renders before

Reply:
hmmm...seems the deploy site is working ok. still curious why this state needs to be moved up

Reply:
if you need some 'defaultState' coming from the parent, that's different